src/util.ts

```diff
@@ -71,20 +74,36 @@ export interface SerdeDecoratorsOptions {
   copyable: boolean
   unionKey?: string
   renameAll?: string
+  parseable: boolean
+  orderable: boolean
+  partialOrderable: boolean
 }
 export function serdeDecorators(options: SerdeDecoratorsOptions): string[] {
   const decorators = []
 
   const deriveMacros = [`Serialize`, `Deserialize`, `Debug`, `Clone`]
-  if (options.partialComparable) {
+
+  // If orderable -> eq needs to also be true (same with partialOrderable -> partialEq)
+  if (options.partialComparable || options.partialOrderable) {
```

Comment:
I would add this check at the options level than.
have the check for either of the options and then add all of them to the options perhaps.